highwinds.py

```diff
@@ -5,18 +5,15 @@
 # Copyright (c) 2016, InnoGames GmbH
 #
 
-from __future__ import print_function
-
+from argparse import ArgumentParser
 import json
 import sys
-import time
+from time import time
```

Comment:
In line 213 `time.strftime()` and `time.gmtime()` are used. You have to import them too.